### PR TITLE
sv-137-testlogout

### PR DESCRIPTION
**Pull request by svoggu**
make logout work by adding logout function

### CODE REVIEW

File: src/server/routers/user-route.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ userRouter.post("/login", function (req, res) {
         const accessToken = jwt.sign({ user }, access_secret);
         res.cookie("jwt", accessToken, {
           httpOnly: true,
-          maxAge: 3600 * 1000,
+          maxAge: 60 * 1000,
```
Comment (brycejjacks) [inline src/server/routers/user-route.ts:44]: Why did you change this... the user went from from being authenticated for 1hr to 1 minute.

**Review by sirpuffin1 — APPROVED** [review]
Okay soooo I'm going to approve it because I think vinson said as long as it doesn't break anything you can approve and just refactor later. BUT I ran it a couple of times and after logging out the user is still able to access the admin locked features. Not a huge deal but yeah I think it's because the token isn't being removed from the user I could totally be wrong though other than that great job :)

File: src/server/routers/user-route.ts [hunk 2]
```diff
@@ -66,11 +66,11 @@ userRouter.post("/login", function (req, res) {
...
-    userRouter.get('/api/check-login', authHandler, (req, res) => {
+    userRouter.get('/check-login', authHandler, (req, res) => {
       res.json({message: 'yes'});
     });
 
-    userRouter.get("/logout", authHandler, function (req, res) {
+    userRouter.get('/logout', authHandler, function (req, res) {
       res.cookie("jwt", "", {
         httpOnly: true,
         maxAge: 60 * 60 * 1000,
```
Comment (brycejjacks) [inline src/server/routers/user-route.ts:76]: i think setting this maxAge doesn't really do anything , also above i think you can use `res.clearCookie("jwt")` jus a fun fact ... you can simply resolve this

File: src/client/src/app/store/reducers/user/user.reducer.ts [hunk 3]
```diff
@@ -17,7 +17,8 @@ export const initialState: State = {
   users: [],
   selectedUser: null,
   loginFailMessage:'',
-  loginUser: JSON.parse(localStorage.getItem('Token') || '{}')
```
Comment (brycejjacks) [inline src/client/src/app/store/reducers/user/user.reducer.ts:20]: this looks destructive ... why are we getting rid of this functionality

**Review by brycejjacks — DISMISSED** [review]
Ok this PR is very very hard to understand lol ... first off i checked out the branch... and you did not even call the logout function that dispatches the logout action... you are simply routing somewhere to a client route that does not exist .. so the page reloads. You need to connect the chain of action, effects reducer... it is also concerning because none of that is happening in this pr .. there is no effects set up along with other stuff missing ... I personally do not want to merge this at all because it touches alot of files and no information is being passed through any of them, so it is impossible for anything to have been tested...... i know this is confusing, but you seemingly achieved a desired result on accident 😬 this needs alot of work still
